File: generated-code/example/java/src/main/java/Runner.java
import java.io.*;

import util.StreamUtil;

public class Runner {
    public static void main(String[] args) throws IOException {
        if (args.length != 2) {
            throw new RuntimeException("Pass input and output as parameters");
        }
        String inputFile = args[0];
        String outputFile = args[1];

        InputStream inputStream = new BufferedInputStream(new FileInputStream(inputFile));
        model.Structure input = model.Structure.readFrom(inputStream);

        OutputStream outputStream = new BufferedOutputStream(new FileOutputStream(outputFile));
        input.writeTo(outputStream);
        outputStream.flush();
        outputStream.close();
    }
}

File: generated-code/example/scala/src/main/scala/Runner.scala
import java.io._
import util.StreamUtil

object Runner extends App {
    if (args.size != 2) {
        throw new Exception("Pass input and output as parameters")
    }
    val inputFile = args(0)
    val outputFile = args(1)

    val inputStream: InputStream = new BufferedInputStream(new FileInputStream(inputFile))
    val input: model.Example = model.Example.readFrom(inputStream)

    val outputStream: OutputStream = new BufferedOutputStream(new FileOutputStream(outputFile))
    input.writeTo(outputStream)
    outputStream.flush()
    outputStream.close()
}